### PR TITLE
omap: switch to kernel 5.4

### PR DESCRIPTION
**Pull request by dhewg**
Runtime tested on beagleblack.

### CODE REVIEW

File: target/linux/omap/config-5.4 [hunk 1]
```diff
@@ -400,14 +409,6 @@ CONFIG_MMC_OMAP_HS=y
 CONFIG_MODULES_USE_ELF_REL=y
 CONFIG_MSDOS_FS=y
 CONFIG_MTD_CMDLINE_PARTS=y
-CONFIG_MTD_M25P80=y
-CONFIG_MTD_NAND=y
```
Comment (olek210) [inline target/linux/omap/config-5.4]: This disables NAND support. Are you sure?
[Commit](https://github.com/torvalds/linux/commit/72c5af00272339af6bbed6fe7275cd731f57be2d#diff-f42d611a391dbc954692f5a7276d7785) renamed this symbol to CONFIG_MTD_RAW_NAND

Comment (dhewg) [reply target/linux/omap/config-5.4]: That wasn't the intention, `make kernel_oldconfig` just didn't prompt for these knobs. Fixed, thanks!

**Comment by dhewg** [comment]
I tried to keep the same nand/sound/panel functionality using new knobs, bit I don't have/use that, so that's untested

**Comment by adrianschmutzler** [comment]
Maybe add a (short/trivial) commit message about why you remove 4.14 support.

**Comment by dhewg** [comment]
Done: "There's support for 5.4 and 4.19 now, remove the 3rd option."

**Comment by adrianschmutzler** [comment]
I've just had a look into #2446. Based on that, only two of four devices were/are usable with 4.19. Nothing has happened since then, and you tested one of them with 5.4, so I think this is good to go.

**Comment by adrianschmutzler** [comment]
Merged, thanks.

**Pull request closed without merging**